avro/src/test/java/com/fasterxml/jackson/dataformat/avro/interop/ApacheAvroInteropUtil.java

```diff
@@ -108,7 +108,7 @@ protected Schema createSchema(Type type, Map<String, Schema> names) {
                     }
                 }
             }
-            if (type instanceof Class<?> && ((Class<?>) type).getSuperclass() != null) {
+            if (type instanceof Class<?> && ((Class<?>) type).getSuperclass() != null && !Enum.class.isAssignableFrom((Class<?>) type)) {
```

Comment:
True; I'm not super-concerned here though because this is just a local patch to the Apache implementation to make it easier to test in a uniform manner as Jackson; All that matters is that the types we use in the unit tests work.